packages/fxa-auth-server/lib/account-delete.ts

```diff
@@ -150,10 +144,18 @@ export class AccountDeleteManager {
     // because obviously we can't retrieve the devices list after!
     try {
       // Check to see if we have an account, this throws if it was deleted
-      await this.fxaDb.account(uid);
+      const account = await this.fxaDb.account(uid);
+
       const devices = await this.fxaDb.devices(uid);
 
       await this.fxaDb.deleteAccount({ uid });
+      this.log.activityEvent({
+        uid,
+        email: account.primaryEmail?.email,
```

Comment:
Q: Is it possible that an account _couldn't_ have a `primaryEmail`? 🤔

Reply:
@vbudhram when the emails table was added did FxA copy the existing account email addresses into it?

Reply:
@chenba they did, primary email should always be set. Don't think this should be an issue.